### ISSUE TITLE
qt5-style-plugin-gtk2 causes immense QT application slowdown on fresh 19.10 installs (but not 19.04 to 19.10 upgrades)

### ISSUE THREAD

**Issue by BaronKrause** [pop-os/gtk-theme#404]
This was reported in a previous bug report https://github.com/pop-os/gtk-theme/issues/394
but it was blocked since it wasn't reproducible. After some testing it seems this issue only happens on fresh 19.10 installs and not systems that had 19.04 and then upgraded to 19.10 and im unsure if blocked bug reports get looked at again when there is new replies.

The issue is the qt5-style-plugin-gtk2 causes immense QT application slowdown on fresh 19.10 installs (but not 19.04 to 19.10 upgrades).

I can give steps to replicate this 100% (tested in VMWare).

Install a fresh copy of Pop 19.10, then install:
"qt5-style-plugins"
"qt5-style-plugin-gtk2"
"qt5ct"

Reboot and open qt5ct and select the gtk2 style and watch it take around 60 seconds to apply. The same will happen with any qt application when you try to open them if you keep it set like that or put QT_QPA_PLATFORMTHEME="gtk2" in your /etc/environment file (or "export QT_QPA_PLATFORMTHEME=gtk2" to ~/.profile).

Now this does not happen to systems that were updated from 19.04. I just performed a fresh 19.04 install in another VM, applied all updates and then let it update to 19.10. Installing those applications and reproducing the steps above does not have any of those issues. So this is something that only applies to fresh 19.10 installs. Maybe there is a dependency that isn't listed but happened to be part of the 19.04 base install? I really have no idea about that but just incase thats relavent I generated a list of apt installed package differences between the fresh 19.10 and the fresh 19.04 to 19.10 system with this script in case it's of any use: https://github.com/lepe/scripts/blob/master/compare_ubuntu_apt.pl:

test@pop-os:~$ ./compare_ubuntu_apt.pl 19.10.lst 19.04-19.10.lst 
----------------------------------
 DIFFERENCES 19.10.lst -> 19.04-19.10.lst
----------------------------------
 `@ gir1.2-mutter-5 : 3.34.1-1ubuntu1pop2 -> 3.34.12ubuntu2pop1~1573148599~19.10~38448ab`
 `@ libmutter-5-0 : 3.34.1-1ubuntu1pop2 -> 3.34.12ubuntu2pop1~1573148599~19.10~38448ab`
 `@ mutter : 3.34.1-1ubuntu1pop2 -> 3.34.12ubuntu2pop1~1573148599~19.10~38448ab`
 `@ mutter-common : 3.34.1-1ubuntu1pop2 -> 3.34.12ubuntu2pop1~1573148599~19.10~38448ab`
----------------------------------
 MISSING IN 19.04-19.10.lst
----------------------------------
 + fwupd-signed : 1.10+1.2.10-1ubuntu2
 + sbsigntool : 0.9.2-2ubuntu1
 + secureboot-db : 1.5
 + system76-acpi-dkms : 1.0.1~1571170639~19.10~d625910
----------------------------------
 MISSING IN 19.10.lst
----------------------------------
 - cpp-8 : 8.3.0-23ubuntu2
 - gcc-8 : 8.3.0-23ubuntu2
 - gcc-8-base : 8.3.0-23ubuntu2
 - libapt-inst2.0 : 1.8.3
 - libapt-pkg5.0 : 1.8.3
 - libdigest-hmac-perl : 1.03+dfsg-2
 - libgcc-8-dev : 8.3.0-23ubuntu2
 - libip4tc0 : 1.6.1-2ubuntu3
 - libjson-c3 : 0.12.1+ds-2
 - libmpx2 : 8.3.0-23ubuntu2
 - libnet-libidn-perl : 0.12.ds-3build1
 - libsub-name-perl : 0.21-1build2
 - libtss2-esys0 : 2.1.0-4
 - libtss2-udev : 2.1.0-4
 - linux-headers-5.0.0-15 : 5.0.0-15.16pop0
 - linux-headers-5.0.0-15-generic : 5.0.0-15.16pop0
 - linux-image-5.0.0-15-generic : 5.0.0-15.16pop0
 - linux-modules-5.0.0-15-generic : 5.0.0-15.16pop0
 - linux-modules-extra-5.0.0-15-generic : 5.0.0-15.16pop0
 - linux-signed-generic : 5.3.0.19.22
 - mlocate : 0.26-3ubuntu3
 - packagekit-tools : 1.1.12-5ubuntu4
 - pop-upgrade : 0.1.0~1572988324~19.10~e8ef7fe
 - tpm2-abrmd : 2.1.1-1
 - tpm2-tools : 3.1.3-2
 - vdpau-va-driver : 0.7.4-7

_Originally posted by @BaronKrause in https://github.com/pop-os/gtk-theme/issues/394#issuecomment-551968047_

**Comment by isantop** [comment]
I'm going to leave this marked as a duplicate of #394

**Comment by BaronKrause** [comment]
> I'm going to leave this marked as a duplicate of #394 so that we keep each issue tracked in a single place.

That makes sense, but will the other one be unblocked, or do blocked submissions still get checked and worked on?
